docs/circuits.md

```diff
@@ -19,12 +19,12 @@ the most common type of `Operation` is a `GateOperation`.
 Let's unpack this.
 
 At the base of this construction is the notion of a qubit.  In
-Cirq, qubits are represented by subclasses of the `QubitId`
-base class. Different subclasses of `QubitId` can be used
+Cirq, qubits are represented by subclasses of the `Qid`
+base class. Different subclasses of `Qid` can be used
```

Comment:
Maybe note that `Qid` is short for "quantum id".

Reply:
Done.